app/assets/javascripts/modules/track-qa-choices.js

```diff
@@ -0,0 +1,83 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (global, GOVUK) {
+  'use strict'
+
+  GOVUK.Modules.TrackQaChoices = function () {
```

Comment:
Could we rename this to something a bit more descriptive? I feel future devs might not understand what "qa" is
```suggestion
  GOVUK.Modules.TrackResponses = function () {
```